base/operators.jl

```diff
@@ -520,6 +536,7 @@ identity(@nospecialize x) = x
 (|)(x::Integer) = x
 xor(x::Integer) = x
 
+const − = -
```

Comment:
This shouldn't be needed.

Reply:
Ah, true, I forgot to remove it from there!